src/install/sql/content_test.sql

```diff
@@ -141,9 +141,10 @@ LOCK TABLES `currency` WRITE;
 
 INSERT INTO `currency` (`id`, `title`, `code`, `is_default`, `conversion_rate`, `format`, `price_format`, `created_at`, `updated_at`)
 VALUES
-	(1,'US Dollar','USD',1,1.000000,'${{price}}','1','2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(2,'Euro','EUR',0,0.600000,'€{{price}}','1','2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(3,'Pound Sterling','GBP',0,0.600000,'{{price}} ₤','1','2022-12-01 12:00:00','2022-12-01 12:00:00');
+	(1,'US Dollar','USD',1,1.000000,'${{price}}','1','1','2022-12-01 12:00:00','2022-12-01 12:00:00'),
+	(2,'Euro','EUR',0,0.600000,'€{{price}}','1','1','2022-12-01 12:00:00','2022-12-01 12:00:00'),
```

Comment:
Same here 
and for pounds 

If we want to add an test case we should add:
https://en.wikipedia.org/wiki/Hungarian_forint

As it should rounded to "whole" forint..

Reply:
That's a remainder of before I cut a step in the calculations. It should be 0.01 ofc, but as I said, before the cart rewrite, I'm not touching this feature, as the cart / order currently calculate VAT in the template, instead of the business logic. This means f.Ex that the calculations would need to be changed in every theme.